### ISSUE TITLE
Notice: add_submenu_page was called incorrectly. (version 5.9.6)

### ISSUE THREAD

**Issue by sainipatala** [AdvancedCustomFields/acf#508]
Upgrading from 5.9.5 to 5.9.6. gives the following notice: Notice: add_submenu_page was called incorrectly. The seventh parameter passed to add_submenu_page() should be an integer representing menu position.

**Comment by texaclou** [comment]
Hello,
we had the same problem with 5.9.6 update.

With acf pro we are calling `acf_add_options_page` to create a **sub** admin page.

`add_submenu_page` is called by ACF when `parent_slug` option is not empty.
In that case,  if the `position` parameter is defined and not a int, the error is throwed.

However in the doc, `position` parameter can be int|string 
[Official doc page](https://www.advancedcustomfields.com/resources/acf_add_options_page/)

```
'position' => 3, //ok
'position' => '3', //throw Notice: add_submenu_page was called incorrectly
```

**Comment by nathanp** [comment]
Thanks @texaclou - this issue was driving me crazy, thanks for the straightforward fix.

**Comment by lgladdy** [comment]
This issue should have been resolved in 5.9.7. Please reopen if you're still having any issues.
